x-pack/legacy/plugins/snapshot_restore/public/app/services/text/text.ts

```diff
@@ -112,6 +112,31 @@ class TextService {
       },
     });
   }
+
+  public getTimeUnitLabel(timeUnit = TIME_UNITS.SECOND, timeValue = '0') {
```

Comment:
wouldnt it be better to have timeValue as a `number` instead of a `string`? also If we're going to show the value of the time its better to have it baked inside this. 

`{timeValue, plural, one {a second} other {# seconds}}`

Reply:
This is used to populate options in a dropdown, so I think `{timeValue, plural, one {minute} other {minutes}}` is correct.

<img width="685" alt="Screen Shot 2019-10-01 at 1 36 14 PM" src="https://user-images.githubusercontent.com/5226211/65985903-c67b5980-e450-11e9-9810-995f407c7df7.png">